### ISSUE TITLE
Auto scroll issue with sticky positioning and full viewport height container

### ISSUE THREAD

**Issue by skflowne** [atlassian/react-beautiful-dnd#866]
## Bug

I have a sidebar in position sticky, inside is a Droppable and inside some Draggables.
This is the CSS of the sticky container :
```
  display: flex;
  flex-direction: column;
  position: sticky;
  top: 70px;
  right: 0;
  height: calc(100vh - 70px);
  max-width: 90px;
```
In there I have a `Droppable` with `overflow: auto` and a `max-height: 84vh`
And in there some `Draggables` that can be reorganised by drag-n-drop. 

### Expected behavior
When I move a `Draggable` near the edge of the `Droppable` container, it would automatically scroll the `Droppable` container with `overflow: auto`

### Actual behavior
When I move a `Draggable` the window starts scrolling, the `Droppable` doesn't scroll at all, even when I reach the end of the window scroll, I still can't make my container scroll.

### What version of `React` are you using?
16.4

### What version of `react-beautiful-dnd` are you running?
9.0.2

### What browser are you using?
Chrome 69

### Demo
[Demo](https://codesandbox.io/s/jjrnj1onj3)
The weird thing is, this seems to be an edge case with the height of the container, if you change 90vh to 50vh, it works as expected.

And in my case, I need to have my sidebar sticky and taking full height of the viewport - my header's height.

**Comment by zachmoore12** [comment]
Any reason your `div` that is positioned `relative` cannot be `100%` height? Adding that gets it to work for me:

https://codesandbox.io/s/pmj8pro3wj

**Comment by alexreardon** [comment]
we currently do not support `position:sticky`. We will be introducing partial support for `position:fixed` in version 10 #838

**Comment by alexreardon** [comment]
It has to do with something no moving on the page when it is expect to. I think for now it is best to say that this is not supported. If improve support for fixed lists then this should fit into that
